packages/block-library/src/post-featured-image/index.php

```diff
@@ -49,22 +49,27 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	if ( $is_link ) {
 		$link_target    = $attributes['linkTarget'];
 		$rel            = ! empty( $attributes['rel'] ) ? 'rel="' . esc_attr( $attributes['rel'] ) . '"' : '';
+		$height         = ! empty( $attributes['height'] ) ? 'style="height:' . esc_attr( $attributes['height'] ) . '"' : '';
```

Comment:
Given that we are escaping a style from the security point of view, we also need to call the function `safecss_filter_attr`. So here we should do esc_attr( safecss_filter_attr( 'height:' . esc_attr( $attributes['height'] ) . '"' ) )`

Reply:
Updated, thank you.